eng/build.sh

```diff
@@ -49,7 +49,7 @@ while (($# > 0)); do
       defaultargs="--test --warnaserror false"
       shift 1
       ;;
-     -includetests|-buildtests)
```

Comment:
This will break current official builds and new CI system. I use -includetests all over the place. Can we leave this option as it doesn't hurt and we can clean it up and update the build definitions on a separate PR?

Reply:
I'm not going to merge these changes before Azure DevOps is enabled and the groovy scripts are removed. The right switch is `buildtests` as `includetests` doesn't make any sense when used together with test, i.e.: `build -test -includetests`. I will search replace all occurrences then.

Reply:
But I thought we would only use `build -includetests` without the -test switch and that would mean build source projects and test projects.

Reply:
If you want to build and test the test projects you can use that switch too. The syntax will look as following:

```
build -> builds all ref and src projects
build -buildtests -> builds all ref, src and test projects
build -test -> runs all unit test projects
build -integrationTest -> runs all integration test projects
build -performanceTest -> runs all performance test projects
build -test -buildtests -> builds and runs all unit test projects (same for integration and perf)
```

In the future there will also be a clean target in arcade which will allow us do the following thing:
``´
build -clean
```